modules/fluid_properties/doc/content/source/userobjects/TabulatedFluidProperties.md

```diff
@@ -109,8 +123,70 @@ FluidProperties UserObject.
 
 !alert note
 All fluid properties read from a file or specified in the input file (and their derivatives with
-respect to pressure and temperature) will be calculated using bicubic interpolation, while all
-remaining fluid properties will be calculated using the provided FluidProperties UserObject.
+respect to pressure and temperature) will be calculated through interpolation, while all
+remaining (or missing) fluid properties will be calculated using the provided `FluidProperties` `UserObject`.
```

Comment:
`FluidProperties` object. Same for occurrences below.

Reply:
should have caught all occurences